packages/@okta/i18n/src/properties/login.properties

```diff
@@ -833,6 +833,12 @@ oie.authenticator.app.method.push.verify.enable.biometrics.point2 = Okta Verify
 oie.authenticator.app.method.push.verify.enable.biometrics.point3 = In Okta Verify, biometrics are enabled for your account
 oie.authenticator.app.method.push.enroll.enable.biometrics = Your organization requires biometrics. To proceed, ensure your device supports biometrics, then add your account and enable biometrics when prompted.
 oie.authenticator.app.method.push.enroll.enable.biometrics.title = Enable biometrics to add an account in Okta Verify
+oie.authenticator.oktaverify.method.totp.verify.enable.biometrics = Your response was received, but your organization requires biometrics. Make sure your device supports biometrics, Okta Verify is up-to-date and biometrics are enabled for your account in Okta Verify, then try again.
```

Comment:
used `Your` 4 times in one message. Seems overused.

Reply:
Feedback noted, but I think if we shifted the framing of the user requirements to reduce the amount of direct language (you, your), I am afraid that the messages will be potentially less concise or unclear